docs/PORTS.md

```diff
@@ -0,0 +1,57 @@
+
+# 29 Dashcam Cars
+
+|Make|Model|Supported Package|ACC|No ACC accel below|No ALC below|Steering Torque|Resume from stop|<a href="##"><img width=2000></a>Hardware Needed<br>&nbsp;|Video|
+|---|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+|Buick|Regal Essence 2018[<sup>4</sup>](#footnotes)|Adaptive Cruise Control (ACC)|openpilot|18 mph|7 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-empty.svg)](##)|<details><summary>Parts</summary><sub>- 1 OBD-II connector<br>- 1 comma 3X<br>- 2 long OBD-C cable<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Buick&model=Regal Essence 2018">Buy Here</a></sub></details>||
+|Cadillac|ATS Premium Performance 2018[<sup>4</sup>](#footnotes)|Adaptive Cruise Control (ACC)|openpilot|18 mph|7 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-empty.svg)](##)|<details><summary>Parts</summary><sub>- 1 OBD-II connector<br>- 1 comma 3X<br>- 2 long OBD-C cable<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Cadillac&model=ATS Premium Performance 2018">Buy Here</a></sub></details>||
+|Chevrolet|Equinox 2019-22|Adaptive Cruise Control (ACC)|openpilot available[<sup>1</sup>](#footnotes)|3 mph|6 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-empty.svg)](##)|<details><summary>Parts</summary><sub>- 1 GM connector<br>- 1 RJ45 cable (7 ft)<br>- 1 comma 3X<br>- 1 comma power v2<br>- 1 harness box<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Chevrolet&model=Equinox 2019-22">Buy Here</a></sub></details>||
+|Chevrolet|Malibu Premier 2017[<sup>4</sup>](#footnotes)|Adaptive Cruise Control (ACC)|openpilot|18 mph|7 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-empty.svg)](##)|<details><summary>Parts</summary><sub>- 1 OBD-II connector<br>- 1 comma 3X<br>- 2 long OBD-C cable<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Chevrolet&model=Malibu Premier 2017">Buy Here</a></sub></details>||
+|Ford|F-150 2023|Co-Pilot360 Active 2.0|openpilot available[<sup>1</sup>](#footnotes)|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|<details><summary>Parts</summary><sub>- 1 Ford Q3 connector<br>- 1 RJ45 cable (7 ft)<br>- 1 comma 3X<br>- 1 comma power v2<br>- 1 harness box<br>- 1 mount<br>- 1 right angle OBD-C cable (1.5 ft)<br><a href="https://comma.ai/shop/comma-3x.html?make=Ford&model=F-150 2023">Buy Here</a></sub></details>||
```

Comment:
I'm an 2023 F-150 XLT with Co-Pilot360 Assist 2.0 owner - see below what the package consists of for a 2023 F-150, you can ignore this comment is you this it's not appropriate. 

![image](https://github.com/commaai/openpilot/assets/8130363/891e26d3-f5a0-449d-9da4-8333ef025bea)

I recommend the following changes:

Change `Co-Pilot360 Active 2.0` to `Co-Pilot360 Assist 2.0`
Change `1 Ford Q3 connector` to `1 Ford Q4 connector`
Change `1 RJ45 cable (7 ft)` to `1 RJ45 cable (1 ft) - or what comes with comma 3X`
Change `1 comma power v2` to `1 comma power v2 (10 ft USB-C extension cable)` - the IPMA is under the steering wheel, close to the ODB-II port.
Change `1 mount` to `1 8degree mount`

in other words I ordered:
- C3X + Q4 harness from comma
- 3D printed a 8degree mount
- ordered a 1ft Ethernet cable and a 10ft USB-C extension cable, maybe a 6.6 ft would be enough but better safe than sorry.

Reply:
Can you open a pull request for these changes, if they're more correct for the US market?

Reply:
Sure, I can create one, once you have your changes merged. If there is a better option, let me know.